candi/bashible/bundles/centos/bootstrap.sh.tpl

```diff
@@ -40,6 +40,13 @@ if [ "${VERSION_ID}" == "8" ] ; then
   alternatives --set python /usr/bin/python3
 fi
 
+for FS_NAME in `egrep -v -e '(^[[:space:]]*$|^#)' /etc/fstab | awk '{ print $1 }'`; do
```

Comment:
It is legal to have weird xfs anywhere but /var/lib/contaierd.

Reply:
It's to complicated to check all variants in script, also, it's unusual case, so, it's simplest method.